src/impl/domain.ts

```diff
@@ -212,12 +212,24 @@ export function getActiveDomain(): StoreDomainInterface {
 /**
  * Get's the unique domain id created for the current context / scope.
  *
- * @returns {(string | undefined)}
+ * Example:
+ * ```
+ * store.getId(); // Returns full id
+ * store.getId(true); // Returns first 8 characters of id. Default false
+ * ```
+ *
+ * @param {boolean} short Set `true` to get short id. Default `false`
+ * @returns {(string | undefined)} Returns full id by default. Set `short` to true to get short id.
  */
-export function getId(): string | undefined {
+export function getId(short: boolean = false): string | undefined {
```

Comment:
IMO this is a bit confusing. Just looking at `getId(true)` doesn't tell you anything. I think we can be more explicit and leave `getId()` as it is. What we can do is create another function `getShortId()` to return the trimmed version of the active domain id.

Thoughts? @kabirbaidhya @silwalanish

Reply:
And we have missing tests.

Reply:
Okay. I will add `getShortId()` to return the trimmed version and leave `getId()` as it is. And the `test`

Reply:
Yes, sounds good. Let's do it.